docs/troubleshooting.md

```diff
@@ -0,0 +1,21 @@
+# Troubleshooting
+
+## Some Image URLs Are Broken With Tachyon Enabled
+
+Prior to WordPress 5.3.1 any image files uploaded with a file name like `example-300x300.jpg` were allowed and unmodified. In order to better support automatic modification of image URLs for features like responsive images the dimensions are removed.
+
+In order to work properly and for performance reasons Tachyon requires all images to not have dimensions as their suffix.
+
+Altis provides a migration command to rename legacy images and make the necessary database updates.
+
+```
+wp media rename-images [--network] [--sites-page=<int>] [--search-replace] [--tables=<tables>] [--include-columns=<columns>]
+```
+
+- `--network` if present will run the process for all sites on the network.
+- `--sites-page` allows you to change the current page of sites. 100 sites will be processed at a time so if you have more you will need to run the command again for each page of sites.
+- `--search-replace` if present will perform a database search and replace process for the updated image names.
```

Comment:
How is this different from the default behavior? Would you pass specific items to search for and replace to this param (e.g. to limit the rename process to a specific batch or naming pattern)? (If so, perhaps a bit of clarity in the docco might be useful along with an example usage.)

Reply:
Or, if the implication is that the database rename doesn't actually happen unless this parameter is passed, that should be noted in the description above.

Reply:
The images themselves and the attachment data is always updated, the `--search-replace` parameter controls whether or not the rest of the database is checked and updated.

I'll call this out specifically in the preamble.